content/ja/docs/reference/glossary/endpoint-slice.md

```diff
@@ -0,0 +1,21 @@
+---
+title: エンドポイントスライス (EndpointSlice)
+id: endpoint-slice
+date: 2024-10-10
+full_link: /docs/concepts/services-networking/endpoint-slices/
+short_description: >
+  Kubernetesリソースでネットワークエンドポイントをグループ化する方法
```

Comment:
If you want to move this PR forward, please translate _**all**_ the sentences properly into Japanese.

Reply:
sure, have changed as you said pls review it once ! thanks.